gcovr/tests/exclude-directories-relative/Makefile

```diff
@@ -1,7 +1,12 @@
 CFLAGS= -fprofile-arcs -ftest-coverage -fPIC
 
 all:
+ifeq ($(OS),Windows_NT)
+	-mkdir build\a
+	-mkdir build\b
+else
```

Comment:
This leads  to error messages like this in Appveyor:

```
make[1]: Entering directory '/c/projects/gcovr-0p8sb/gcovr/tests/exclude-directories-relative'
mkdir build\a
mkdir: cannot create directory 'build\\a': No such file or directory
```

However, because of the leading `-` in the recipe line the error is ignored and the test fails at a later stage.

Is this because our Appveyor setup uses MSYS?